exchanges/lbank/lbank_test.go

```diff
@@ -0,0 +1,341 @@
+package lbank
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/thrasher-/gocryptotrader/config"
+	"github.com/thrasher-/gocryptotrader/currency"
+	exchange "github.com/thrasher-/gocryptotrader/exchanges"
+	log "github.com/thrasher-/gocryptotrader/logger"
+)
+
+// Please supply your own keys here for due diligence testing
+const (
+	testAPIKey    = ""
+	testAPISecret = ""
+)
```

Comment:
Add const canManipulateOrders = false, so that in the event of any sort of order management function being tested does not execute without the prior authority of the developer. This could be submit order or cancel order functions.